### ISSUE TITLE
Issue with "Web accessibility for seizures and physical reacti…": Asterisks all around the page

### ISSUE THREAD

**Issue by PythonCoderAS** [mdn/content#11222]
MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Seizure_disorders

#### What information was incorrect, unhelpful, or incomplete?

Once again, this page is one of the pages containing blocks of code such as:

> Use the **{{HTMLElement('link')}}** element, alongside with and together with the attributes of **`rel="alternate stylesheet"`** and for title,** `title="…"`** in the **{{HTMLElement('HEAD')}}** section of the webpage.

```markdown
Use the **{{HTMLElement('link')}}** element, alongside with and together with the attributes of **`rel="alternate stylesheet"`** and for title,** `title="…"`** in the **{{HTMLElement('HEAD')}}** section of the webpage.
```

This is probably related to https://github.com/mdn/content/pull/11202 and https://github.com/mdn/yari/issues/5074.

#### Specific section or headline?

Multiple, N/A

#### What did you expect to see?

Properly bolded elements

#### Did you test this? If so, how?

N/A

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/accessibility/seizure_disorders`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/Seizure_disorders
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/seizure_disorders/index.md
* Last commit: https://github.com/mdn/content/commit/8f164b688448a602df794f9094e9d4d039887639
* Document last modified: 2021-12-02T19:40:14.000Z

</details>

**Comment by cw118** [comment]
For the example given above, I'd say it's more a matter of whitespace than of issues with Markdown conversion (I've seen similar situations before). Take the two examples below: 
- Space between asterisks and backtick (markdown is ** \`title="…"\`\**): ** `title="…"`**
- No space between asterisks and backtick (markdown is \**\`title="…"\`**): **`title="…"`**

Notice that in the second example, which doesn't have a space between the left-flank asterisks and the first backtick, renders correctly.

**Comment by cw118** [comment]
I'd be happy to work on this issue and #11203.

**Comment by hamishwillee** [comment]
So the intent was generally to have bold around whatever is marked up with `**`. Honestly, I'd just fix this by removing that markup - there doesn't appear to be much need for it to be bold.

**Comment by cw118** [comment]
> So the intent was generally to have bold around whatever is marked up with `**`. Honestly, I'd just fix this by removing that markup - there doesn't appear to be much need for it to be bold.

Will do, thanks again!
